src/components/action-popover/action-popover-item.d.ts

```diff
@@ -8,6 +8,8 @@ export interface ActionPopoverItemProps {
   disabled?: boolean;
   onClick: () => void;
```

Comment:
As per Sam's comment. If `onClick` is no longer a required prop, this will need to become `onClick?:`